src/tiles/Tiles.js

```diff
@@ -47,7 +47,8 @@ export const Tiles = ({
       }
     }, 300);
 
-    setIsFlipped(!isFlipped);
+    if(tileName!="Color Shuffle")
+      setIsFlipped(!isFlipped);
```

Comment:
I get that you're keeping the tile side up for Color Shuffle but since it has the same text in `tileName` and `content`, we can either keep the fontWeight bold on both the sides for consistency

Reply:
Actually after you click a tile the `isFlipped` is getting complemented so true changes to false and false changes to true! So here for all tiles it is done so that we can shift between the `tileName` and the `content` but for the `color shuffle` tile both content and tile name are same even the styles, cauz the bold font size have to remain there even after click.

So here for all the tiles except for the shuffle tile the `isFlipped` value will get complemented and the flip side will be shown, but for shuffle tile even if you click the display content wont be changed but all its effects would remain same. 


Is it fine or should I look for something else!?

Reply:
@siddhi-gh I would recommend bold! After my previous commit, I think its bold, it wont change after clicking.

Reply:
I'd say this should work for now. We can reiterate over the `Color Shuffle` tile functionality later.